### PR TITLE
Added RTC_CENTER property support to b3dm

### LINKED ISSUE #6488: Add RTC_CENTER support for b3dm
`RTC_CENTER` will soon be a part of the `b3dm` format and we should add support for it in Cesium.

Likely [CESIUM_RTC](https://github.com/KhronosGroup/glTF/tree/master/extensions/1.0/Vendor/CESIUM_RTC) and `RTC_CENTER` will share the same implementation.

Will need to think about how this interacts with [relocating a tileset](https://github.com/AnalyticalGraphicsInc/cesium/issues/6394)

See https://github.com/AnalyticalGraphicsInc/3d-tiles/issues/263 and https://github.com/AnalyticalGraphicsInc/3d-tiles/pull/301

### PR DESCRIPTION
**Pull request by ggetz**
Fixes #6488

Reflects the spec changes (https://github.com/AnalyticalGraphicsInc/3d-tiles/issues/263) as part of 3D Tiles 1.0.

Part of #6697

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Signed CLA is on file.

@ggetz, thanks for the pull request! Maintainers, we have a signed CLA from @ggetz, so you can review this at any time.

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

File: Source/Scene/Batched3DModel3DTileContent.js [hunk 1]
```diff
@@ -346,6 +355,14 @@ define([
             primitive : tileset
         };
 
+        content._rtcCenterTransform = Matrix4.IDENTITY.clone();
```
Comment (lilleyse) [inline Source/Scene/Batched3DModel3DTileContent.js]: Prefer the static function form: `Matrix4.clone(Matrix4.IDENTITY)`

File: Specs/Data/Cesium3DTiles/Batched/BatchedWithRtcCenter/tileset.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
+{
+  "asset": {
+    "version": "0.0"
```
Comment (lilleyse) [inline Specs/Data/Cesium3DTiles/Batched/BatchedWithRtcCenter/tileset.json]: Change version to 1.0

File: Specs/Scene/Batched3DModel3DTileContentSpec.js [hunk 3]
```diff
@@ -371,6 +374,31 @@ defineSuite([
...
+    it('transforms model positions by RTC_CENTER property in the features table', function() {
+        return Cesium3DTilesTester.loadTileset(scene, withRtcCenterUrl).then(function(tileset) {
+            Cesium3DTilesTester.expectRenderTileset(scene, tileset);
+
+            var rtcTransform = tileset._root._content._rtcCenterTransform;
+            expect(rtcTransform).toEqual(Matrix4.fromTranslation(new Cartesian3(0.1, 0.2, 0.3)));
+
+            var expectedModelTransform = Matrix4.multiply(tileset._root.transform, rtcTransform, new Matrix4());
+            expect(tileset._root._content._contentModelMatrix).toEqual(expectedModelTransform);
+            expect(tileset._root._content._model._modelMatrix).toEqual(expectedModelTransform );
```
Comment (lilleyse) [inline Specs/Scene/Batched3DModel3DTileContentSpec.js]: Remove space: `(expectedModelTransform );`

**Comment by ggetz** [comment]
Thanks @lilleyse, all cleaned up!

**Comment by lilleyse** [comment]
👍

**Pull request merged**